lib/jekyll-mentions.rb

```diff
@@ -18,7 +18,7 @@ def generate(site)
 
     def mentionify(page)
       return unless page.content.include?('@')
-      page.content = @filter.mention_link_filter(page.content)
+      page.content = @filter.mention_link_filter(page.content, '/', nil, /[a-zA-Z0-9][a-zA-Z0-9-]*/)
```

Comment:
Exactly! I fixed to use `HTML::Pipeline::MentionFilter::UsernamePattern` with `i` flag instead of `/[a-zA-Z0-9][a-zA-Z0-9-]*/`.